### PR TITLE
[MIG][10.0] web_tree_many2one_clickable

### PR DESCRIPTION
**Pull request by SodexisTeam**
Minor changes only
cc: @pedrobaeza @antespi @Yajo

### CODE REVIEW

File: web_tree_many2one_clickable/migrations/9.0.1.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```
Comment (pedrobaeza) [inline web_tree_many2one_clickable/migrations/9.0.1.0.0/pre-migration.py]: Please remove this file

Comment (atchuthan) [reply web_tree_many2one_clickable/migrations/9.0.1.0.0/pre-migration.py]: @pedrobaeza removed the migration

**Comment by pedrobaeza** [comment]
Better to remove the commit that add it (it's separated) and then the repo final size will be smaller

**Comment by atchuthan** [comment]
@pedrobaeza removed the commit

File: web_tree_many2one_clickable/__manifest__.py [hunk 2]
```diff
@@ -2,16 +2,18 @@
...
 {
     "name": "Clickable many2one fields for tree views",
     "summary": "Open the linked resource when clicking on their name",
-    "version": "9.0.1.0.0",
+    "version": "10.0.1.0.0",
     "category": "Hidden",
     "website": "https://odoo-community.org/",
     "author": "Therp BV, "
               "Tecnativa, "
+              "Sodexis, "
```
Comment (pedrobaeza) [inline web_tree_many2one_clickable/__manifest__.py]: The changes you have made don't make that you should put it on the author list

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Tried on runbot and working as expected, but the author thing is blocking IMO

File: web_tree_many2one_clickable/__manifest__.py (hunk 2 above)
Comment (yajo) [reply web_tree_many2one_clickable/__manifest__.py]: I don't think contributions should always be counted as number of LOC. Migrating an addon involves time (test it, fix it, send PR...). IMHO it's fair to let the contributors be honoured by these little credits.

**Review by yajo — APPROVED** [review]

File: web_tree_many2one_clickable/__manifest__.py (hunk 2 above)
Comment (pedrobaeza) [reply web_tree_many2one_clickable/__manifest__.py]: @Yajo, I repeat, it's not the case (see diff), and this is not contribution, it's authorship.

**Review by tarteo — APPROVED** [review]
Thanks!

**Comment by pedrobaeza** [comment]
Please change authorship and I'll merge.

**Comment by dvdhinesh** [comment]
@pedrobaeza 

@atchuthan is off for a week. He will continue to work on this by next week. Thanks.

**Comment by atchuthan** [comment]
corrections done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**